common/predictive-text/unit_tests/in_browser/cases/worker-trie-integration.js

```diff
@@ -14,6 +14,13 @@ describe('LMLayer using the trie model', function () {
                                                   // the WebWorker boundary, so we should be generous here.
       var lmLayer = new LMLayer(helpers.defaultCapabilities);
 
+      // As noted in worker-dummy-integration as well.
+      if(navigator.userAgent.indexOf('MSIE') !== -1 || navigator.appVersion.indexOf('Trident/') > -1) {
+        // Our wordbreaking uses the IE-unsupported .codePointAt() function.
+        console.warn("Bypassing wordbreak test on IE.");
+        return;
```

Comment:
Please change this line to:

```suggestion
        this.skip();
```

A return from the test function marks the test as _passed_, when in reality, we want to by-passing the test. I want to avoid a head-scratchy situation in the future, if we see witness a bug in this code, then look at the test logs and say "but the tests pass??!?!??!!"

By the way, with `this.skip()`, you don't need a `return`, because `this.skip` [halts execution of the test](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d51cebc037114391e2a36a45644a7eb990435ee5/types/mocha/index.d.ts#L1153-L1156).

Reply:
Good to know that's a thing.  I'll change the one in worker-dummy-integration to match as well; it was using `return` too.